crates/matrix-sdk/src/sliding_sync.rs

```diff
@@ -967,8 +967,6 @@ impl SlidingSync {
                             *self.extensions.lock().unwrap() = self.sent_extensions.lock().unwrap().take();
 
                             debug!(?self.extensions, "Resetting view stream");
-
-                            continue
```

Comment:
@gnunicorn wdyt about this? Does it make sense to let consumers of the sliding sync stream know about the unknown-pos so they can take additional action? All the other things I find a bit ugly about this PR / commit are contained to the bindings, so I'm less hesitant about merging that to `main`.